ui/app/components/tool-actions-form.js

```diff
@@ -4,11 +4,12 @@ import { inject as service } from '@ember/service';
 import Component from '@ember/component';
 import { setProperties, computed, set } from '@ember/object';
 import { addSeconds, parseISO } from 'date-fns';
+import { A } from '@ember/array';
 
 const DEFAULTS = {
   token: null,
   rewrap_token: null,
-  errors: [],
+  errors: A(),
```

Comment:
is this standard now for all default arrays?

Reply:
It is one approach when initializing a property to an array. Another is to use the `init` hook. Here's the [link to the eslint rule](https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/avoid-leaking-state-in-ember-objects.md).